p2p/src/authenticated/lookup/channels.rs

```diff
@@ -64,14 +83,57 @@ impl<P: PublicKey> crate::Sender for Sender<P> {
             return Err(Error::MessageTooLarge(message_len));
         }
 
-        // Wait for messenger to let us know who we sent to
+        // Get the concrete list of peers to send to
+        let peers: Vec<Self::PublicKey> = match recipients {
```

Comment:
Is there a way to check if rate_limited before using (I suppose we could just tailor our logic elsewhere to eagerly attempt send rather than checking then send)?

Reply:
Discussed offline; opting to not fork `governor` to add this ability nor add a `bypass_limit` flag to the `Sender::send` API.